devtools/common/src/main/java/io/quarkus/templates/rest/BasicRest.java

```diff
@@ -154,6 +156,11 @@ private void createDockerIgnore() throws IOException {
         generate("templates/dockerignore.ftl", context, docker, "docker ignore");
     }
 
+    private void createGitIgnore() throws IOException {
+        File gitignore = new File(mkdirs(projectRoot), ".gitignore");
+        generate("templates/gitignore.ftl", context, gitignore, "docker ignore");
```

Comment:
I think we have a typo: `resourceType` should be `"git ignore"` and not `"docker ignore"`.